src/training.rs

```diff
@@ -490,10 +495,32 @@ mod tests {
         let items = anki21_sample_file_converted_to_fsrs();
         let (pre_trainset, trainsets, testset) = split_data(items.clone(), n_splits);
         let items = [pre_trainset.clone(), testset.clone()].concat();
+        dbg!(pre_trainset.len());
+        dbg!(testset.len());
         let average_recall = calculate_average_recall(&items);
         dbg!(average_recall);
         let initial_stability = pretrain(pre_trainset, average_recall).unwrap();
         dbg!(initial_stability);
+        let pretrained_parameters: Vec<f32> = initial_stability
+            .into_iter()
+            .chain(DEFAULT_PARAMETERS[4..].iter().copied())
+            .collect();
+        let pretrained_fsrs = FSRS::new(Some(&pretrained_parameters)).unwrap();
+        let pretrained_rmse = pretrained_fsrs
+            .evaluate(testset.clone(), |_| true)
+            .unwrap()
+            .rmse_bins;
+        let default_fsrs = FSRS::new(Some(&[])).unwrap();
+        let default_rmse = default_fsrs
+            .evaluate(testset.clone(), |_| true)
+            .unwrap()
+            .rmse_bins;
+        let (best_parameters, best_rmse) = if pretrained_rmse < default_rmse {
```

Comment:
I think that these tests are no longer needed.

Reply:
Good catch

Reply:
It is just used for test.

Reply:
Any other feedback?

Reply:
> Any other feedback?

Nothing more from me.